pkg/virt-controller/watch/vmi.go

```diff
@@ -324,7 +326,8 @@ func (c *VMIController) updateStatus(vmi *virtv1.VirtualMachineInstance, pods []
 				// so let's hand over the vmi too
 				vmiCopy.Status.Interfaces = []virtv1.VirtualMachineInstanceNetworkInterface{
 					{
-						IP: pod.Status.PodIP,
+						IP:   pod.Status.PodIP,
+						Name: podInterface0_name,
```

Comment:
You should use here `volumes[x].Name` or `devices.interfaces[x].Name` of the volume/interface which is connected to the pod volume. So if you consider this Spec:

```yaml
spec:
  domain:
    devices:
      interfaces:
      - name: testnet
        bridge: {}
  networks:
  - name: testnet
    pod: {}
```

The status should then look like this:

```yaml
status:
  interfaces:
    - name testnet
      ip: ...
      mac: ...
```

`testnet` will also be set as alias on the domain xml, so you can always rely on that identifier throughout the code.